src/lpython/python_evaluator.h

```diff
@@ -59,10 +71,10 @@ class PythonCompiler
     Allocator al;
 #ifdef HAVE_LFORTRAN_LLVM
     std::unique_ptr<LLVMEvaluator> e;
-    int eval_count;
+    size_t eval_count;
```

Comment:
This is passed as argument to `python_ast_to_asr()`. The call to `python_ast_to_asr()` is also used by other backends. So, this should be declared outside of `#ifdef HAVE_LFORTRAN_LLVM`.